code/mobile/android/PhoneVR/app/src/main/cpp/alvr_main.cpp

```diff
@@ -17,6 +19,11 @@ using namespace nlohmann;
 
 uint64_t HEAD_ID = alvr_path_string_to_id("/user/head");
 
+// TODO: Make this configurable.
```

Comment:
I think you need to make another cpp files because arcore itself seems to have a lot more glue code to use it.